components/Buttons/examples/supplemental/ButtonsTypicalUseSupplemental.m

```diff
@@ -130,7 +131,8 @@ - (void)layoutButtonsInRange:(NSRange)range around:(CGFloat)centerX {
     UILabel *label = self.labels[i];
 
     button.frame = CGRectOffset(button.frame, 0, verticalCenterY);
-    label.frame = CGRectOffset(label.frame, 0, verticalCenterY);
+    label.frame = MDCRectAlignToScale(CGRectOffset(label.frame, 0, verticalCenterY),
+                                      [UIScreen mainScreen].scale);
```

Comment:
We may also want to consider avoiding `mainScreen` altogether and using [UIView.window.screen](https://developer.apple.com/documentation/uikit/uiwindow/1621597-screen) instead.